krux_boto/cli.py

```diff
@@ -25,9 +25,9 @@
 
 class Application(krux.cli.Application):
 
-    def __init__(self, name=NAME):
+    def __init__(self, name=NAME, *args, **kwargs):
         # Call to the superclass to bootstrap.
-        super(Application, self).__init__(name=name)
+        super(Application, self).__init__(name=name, *args, **kwargs)
```

Comment:
We can pass unlimited number of arguments to `krux.cli.Application` or `krux_boto.cli.Application` this way. However, these classes are not doing anything with it. So, is there any point in doing this? Can't you just overload the `__init__()` in your subclass and call the existing constructor for the superclass?